package.json

```diff
@@ -16,7 +16,7 @@
     "@vuepress/client": "2.0.0-beta.61",
     "@vuepress/plugin-google-analytics": "2.0.0-beta.61",
     "vue": "^3.2.47",
-    "vuepress": "2.0.0-beta.61",
+    "vuepress": "^2.0.0-beta.61",
```

Comment:
We don't want to add the caret since it is still in beta. When they release a stable version we will prefer the latest minor release.